src/containers/ttree.d

```diff
@@ -133,74 +169,113 @@ struct TTree(T, Allocator = Mallocator, bool allowDuplicates = false,
 	 */
 	bool remove(T value, void delegate(T) cleanup = null)
```

Comment:
Figure out what should be done here when duplicates are enabled. This should probably return `size_t` like `insert`.

Reply:
* Arbitrarily decided that `remove` would only remove a single element at a time
* documented this behavior 
* added a test